src/lib.rs

```diff
@@ -78,7 +78,7 @@ pub use identity::{
     app::{FullId as AppFullId, PublicId as AppPublicId},
     client::{FullId as ClientFullId, PublicId as ClientPublicId},
     node::{FullId as NodeFullId, PublicId as NodePublicId},
-    PublicId,
+    FullId as FullIdentity, PublicId,
```

Comment:
If we do keep the `FullId` enum, then can we just export it as `FullId` rather than `FullIdentity` just for consistency?